data/filedefs/filetypes.verilog

```diff
@@ -37,7 +37,7 @@ mime_type=text/x-verilog
 
 # these characters define word boundaries when making selections and searching
 # using word matching options
-#wordchars=_abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789
+wordchars=_abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789$
```

Comment:
I just went through other filetypes and none of them defines `wordchars` (all of them have it commented-out). I think it's meant to be an example that can be overridden by users. So maybe best to do the same in this case too.

Reply:
OK, as you wish :)

I did notice that a few of them (R, CSS, Smalltalk, Raku) use a different list though, although it's commented out as you said.  In any case, this only seems to affect what happens when you double-click a word or select "Find only whole words", but for instance Ctrl-click doesn't work if I declare a signal called `foo$bar`.  Not too useful overall I guess.
I'll leave it commented out for consistency with other formats as you request, but if it's OK I'll leave the $ in there.

(This field doesn't seem to be widely used across formats though.  For example LaTeX doesn't touch it, despite of `_` not being a wordchar and `@` often being one, at least in packages.)